src/Services/privateApiService.js

```diff
@@ -52,5 +52,15 @@ export const privateRequestPut = async(path,id,body) => {
 
     return data;
 };
-
+export const onPrivateDelete = async(url,id) =>{
```

Comment:
please use only privateDelete instead of onPrivateDelete

Reply:
changed